backend/parkinglot/src/main/java/touk/demo/parkinglot/model/dto/FreeSpotInfo.java

```diff
@@ -0,0 +1,48 @@
+package touk.demo.parkinglot.model.dto;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import java.util.Date;
+import org.springframework.format.annotation.DateTimeFormat;
+
+public class FreeSpotInfo {
+
+  private String status;
+  private int number;
+
+  @JsonFormat(pattern = "yyyy-MM-dd@HH:mm:ss")
```

Comment:
a jak postanowisz zmienić format to we wszystkich miejscach go będziesz musiał poprawić, nie da się czegoś wyjąć przed nawias?

Reply:
Wydawało mi się że jest to jedyny sposób na zadanie formatu.